UI/Reports/filters/batches.html

```diff
@@ -1,27 +1,23 @@
 <?lsmb
     PROCESS "elements.html";
     PROCESS 'report_base.html';
-    action = 'list_batches';
 
-    FOR CLS IN batch_classes;
-        CLS.text = CLS.class;
-        CLS.value = CLS.id;
-    END;
+    # Empty item at top of batch class list - Allow any class
```

Comment:
This can actually be achieved using the "default_blank" option for the SELECT block (see https://github.com/ledgersmb/LedgerSMB/blob/master/UI/lib/elements.html#L249)

Reply:
Thanks. Changed in commit 402654bcf9e34fedb9f349efdc68c67862264764